app/src/main/java/com/example/concordia_campus_guide/Activities/MainActivity.java

```diff
@@ -53,6 +57,10 @@ protected void onCreate(Bundle savedInstanceState) {
         swipeableInfoCard = BottomSheetBehavior.from(infoCard);
 
         locationFragment = (LocationFragment) getSupportFragmentManager().findFragmentById(R.id.locationFragment);
+
+        AppDatabase appDB = AppDatabase.getInstance(this);
+        List<Shuttle> jas = appDB.shuttleDao().getScheduleByCampusAndDay("SGW", "Friday");
+        String gs = "Asda" ;
```

Comment:
I don't think this line is doing anything here. right? If yes, can you remove it?